src/service/project.ts

```diff
@@ -88,3 +91,105 @@ export const isLikedByUser = async (projectId: number, userId: number) => {
   });
   return Boolean(result);
 };
+
+export const createProject = async (
+  user_id: number,
+  createArgs: ProjectCreateArgs
+) => {
+  return await prisma.project.create({
+    data: {
+      owner_id: user_id,
+      name: createArgs.name,
+      introduction: createArgs.introduction,
+      description: createArgs.description,
+      link: createArgs.link,
+      logo: createArgs.logo,
+      start_at: createArgs.start_at,
+      end_at: createArgs.end_at,
+      ProjectSkill: {
+        create: createArgs.skills.map((skill) => {
+          return { name: skill.name };
+        }),
+      },
+      ProjectMember: {
+        create: createArgs.members.map((m) => {
+          return { member: m.member_id, role: m.role };
+        }),
+      },
+      ProjectField: {
+        create: createArgs.fields.map((field) => {
+          return { name: field.name };
+        }),
+      },
+      ProjectImage: {
+        create: createArgs.images.map((image) => {
+          return { link: image.link, order: image.order };
+        }),
+      },
+    },
+  });
+};
+
+export const updateProject = async (
+  id: number,
+  updateArgs: ProjectCreateArgs
+) => {
+  return await prisma.project.update({
+    where: {
+      id: id,
+    },
+    data: {
+      name: updateArgs.name,
+      introduction: updateArgs.introduction,
+      description: updateArgs.description,
+      link: updateArgs.link,
+      logo: updateArgs.logo,
+      start_at: updateArgs.start_at,
+      end_at: updateArgs.end_at,
+      ProjectSkill: {
+        set: updateArgs.skills,
+      },
+      ProjectMember: {
+        set: updateArgs.members,
+      },
+      ProjectField: {
+        set: updateArgs.fields,
+      },
+      ProjectImage: {
+        set: updateArgs.images,
+      },
+    },
+  });
+};
+
+export const deleteProject = async (id: number) => {
+  return await prisma.project.delete({
+    where: {
+      id: id,
+    },
+  });
+};
+
+export const createLikeProject = async (
+  user_id: number,
+  project_id: number
+) => {
+  await prisma.proejctLike.create({
+    data: {
+      user_id: user_id,
+      project_id: project_id,
+    },
+  });
+};
+
+export const deleteLikeProject = async (
+  user_id: number,
+  project_id: number
+) => {
+  await prisma.proejctLike.deleteMany({
```

Comment:
하나만 지우는 것이기 때문에 굳이 deleteMany를 쓸 이유는 없겠네요 :D
```suggestion
  await prisma.proejctLike.delete({
```